onfido-java/src/test/java/com/onfido/integration/TestsHelper.java

```diff
@@ -0,0 +1,209 @@
+package com.onfido.integration;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.UUID;
+import java.util.List;
+import java.util.Arrays;
+
+import com.onfido.Onfido;
+import com.onfido.Onfido.Builder;
+import com.onfido.JsonObject;
+import com.onfido.exceptions.OnfidoException;
+
+import com.onfido.models.Applicant;
+import com.onfido.models.Check;
+import com.onfido.models.Document;
+import com.onfido.models.Location;
+import com.onfido.models.Webhook;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+class TestsHelper {
+
+  static private String apiToken = System.getenv("ONFIDO_API_TOKEN");
+  static protected String sampleApplicantId = System.getenv("ONFIDO_SAMPLE_APPLICANT_ID");
+
+  private Onfido.Builder builder = Onfido.builder();
+  protected Onfido onfido;
+
+  private MockWebServer server;
+  private String request;
+
+  public TestsHelper() {
```

Comment:
Yeah, all the test subclasses inheriting from `TestHelper` are implicitly calling it (otherwise `onfido` field would not be initialized when mocking is disabled).